### PR TITLE
Task/enable telemetry in tst

### PR DESCRIPTION
**Pull request by jobala**
## Overview

1. Enable telemetry in -tst
2. Specify the source of a telemetry event ie GE v3 or GE v4

### CODE REVIEW

File: src/telemetry/telemetry.ts [hunk 1]
```diff
@@ -9,11 +9,13 @@ class Telemetry implements ITelemetry {
   private reactPlugin: any;
 
   constructor() {
+    const { mscc } = (window as any);
```
Comment (thewahome) [inline src/telemetry/telemetry.ts]: this value is never used. Do we need to declare it here?

File: src/telemetry/telemetry.ts [hunk 2]
```diff
@@ -9,11 +9,13 @@ class Telemetry implements ITelemetry {
...
+
     this.reactPlugin  = new ReactPlugin();
     this.config = {
-      instrumentationKey: process.env.REACT_APP_INSTRUMENTATION_KEY,
+      instrumentationKey: this.getInstrumentationKey(),
       disableExceptionTracking: true,
-      disableTelemetry: this.areWeInDev() ? false : true,
+      disableTelemetry: this.getInstrumentationKey() ? false : true,
```
Comment (thewahome) [inline src/telemetry/telemetry.ts]: From what I understand, we disable Telemetry when we do not find an instrumentation key. Right?
However it seems that even without a key in the env file, it does not disable telemetry.

I also noticed that if the key is empty, it still does not disable

Comment (jobala) [reply src/telemetry/telemetry.ts]: That is interesting, following up on your DM to learn more about why telemetry is not disabled.

Comment (jobala) [reply src/telemetry/telemetry.ts]: Resolving this.

We have to stop and start the server for it to pick up changes from the .env file

**Pull request merged**